README.md

```diff
@@ -98,6 +98,21 @@ Convert an observation to an OSM type `node`.
 
 Retrieves the current device's unique ID (string).
 
+### `mapeo.getDatabaseStatus(cb)`
+
+Retrieves the status of the database, returns an array of Device IDs and their download status.
+
+```
+{ 
+  devices: [{
+    id: DeviceID (String[64]),
+    sofar: Number,
```

Comment:
I think that `dbDownloaded` would be clearer to a developer than `sofar`. If we do support sparse and allow intentional removal of data from a device (either for disk space or security reasons) the concept of "I have downloaded X records out of Y total" makes more sense to me.

Reply:
`sofar` is currently what is passed in through progress so I wanted to keep it uniform.

What about `database` or `feeds` instead of `devices` ?

Reply:
going with `feeds`